pristine-zen-garden.html

```diff
@@ -1,157 +1,145 @@
-  </li>         </ul>
-</nav>
-</div>
-
-<div class="design-archives" id="design-archives">
-<h3 class="archives">Archives:</h3>
-<nav role="navigation">
-<ul>
-<li class="next">
-<a href="/214/page3">
-Next Designs <span class="indicator">&rsaquo;</span>
-</a>
-</li>
-<li class="previous">
-<a href="/214/page1/">
-<span class="indicator">&lsaquo;</span> Previous Designs              </a>
-</li>
-<li class="viewall">
-<a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden.">
-View All Designs              </a>
-</li>
-</ul>
-</nav>
-</div>
-
-<div class="zen-resources" id="zen-resources">
-<h3 class="resources">Resources:</h3>
-<ul>
-<li class="view-css">
-<a href="/214/214.css" title="View the source CSS file of the currently-viewed design.">
-View This Design&#8217;s <abbr title="Cascading Style Sheets">CSS</abbr>            </a>
-</li>
-<li class="css-resources">
-  <a href="http://www.mezzoblue.com/zengarden/resources/" title="Links to great sites with information on using CSS.">
-<abbr title="Cascading Style Sheets">CSS</abbr> Resources           </a>
-</li>
-<li class="zen-faq">
-  <a href="http://www.mezzoblue.com/zengarden/faq/" title="A list of Frequently Asked Questions about the Zen Garden.">
-<abbr title="Frequently Asked Questions">FAQ</abbr>           </a>
-</li>
-<li class="zen-submit">
-  <a href="http://www.mezzoblue.com/zengarden/submit/" title="Send in your own CSS file.">
-Submit a Design           </a>
-</li>
-<li class="zen-translations">
-<a href="http://www.mezzoblue.com/zengarden/translations/" title="View translated versions of this page.">
-Translations            </a>
-</li>
-  </ul>
-  </div>
-</div>
-</aside>
-
-
-</div>
-
-</body>
+   <head>
+      <meta charset="utf-8">
+      <title>CSS Zen Garden: The Beauty of CSS Design Test</title>
+      <link rel="stylesheet" media="screen" href="/214/214.css?v=8may2013">
+      <link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta name="author" content="Dave Shea">
+      <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+      <meta name="robots" content="all">
+   </head>
+   <body id="css-zen-garden">
+      <div class="page-wrapper">
+         <section class="intro" id="zen-intro">
+            <header role="banner">
+               <h1>CSS Zen Garden</h1>
+               <h2>The Beauty of <abbr title="Cascading Style Sheets">CSS</abbr> Design</h2>
+            </header>
+            <div class="summary" id="zen-summary" role="article">
+               <p>A demonstration of what can be accomplished through <abbr title="Cascading Style Sheets">CSS</abbr>-based design. Select any style sheet from the list to load it into this page.</p>
```

Comment:
for <p> i would have it more like this to make it easier to read
<p>
      text
</p>

Reply:
Hello there! Thanks for reviewing. I'm still getting used to knowing what are your edits and what are mine, and I have to refer to my master commit and my tidying commit so I can compare. So just to make sure, your edits are colored green? Excuse my lack of knowledge. Thanks.

Reply:
all good it takes time to get use to this, but I didnt make any changes I just made comments and suggested changes. because most of yours can be done multiple ways I just suggested the way I did it.